airbyte-integrations/connectors/source-intercom/unit_tests/unit_test.py

```diff
@@ -0,0 +1,339 @@
+#
```

Comment:
this file was reset by reverting [this PR](https://github.com/airbytehq/airbyte/pull/23013)